galaxy/templates/deployment-web.yaml

```diff
@@ -30,7 +30,7 @@ spec:
           command: ['sh', '-c', 'until nc -z -w3 {{ template "galaxy-postgresql.fullname" $ }} 5432; do echo waiting for galaxy-postgres service; sleep 1; done;']
         - name: {{ .Chart.Name }}-init-mounts
           image: alpine:3.7
-          command: ['sh', '-c', 'chown -R 101:101 /galaxy/server/database && install -o 101 -g 101 /galaxy/server/config/integrated_tool_panel.xml /galaxy/server/config/editable/integrated_tool_panel.xml']
+          command: ['sh', '-c', 'chown -R 101:101 /galaxy/server/database && install -o 101 -g 101 /galaxy/server/config/integrated_tool_panel.xml /galaxy/server/config/editable/integrated_tool_panel.xml; if [ ! -f "/galaxy/server/database/config/editable_shed_tool_conf.xml" ]; then mkdir -p /galaxy/server/database/config && echo "<?xml version=\"1.0\"?>\n<toolbox tool_path=\"database/shed_tools\">\n</toolbox>" > /galaxy/server/database/config/editable_shed_tool_conf.xml; fi']
```

Comment:
Since this entire init container is repeated, it seems like a ripe candidate to be moved into a template?

Reply:
Done in https://github.com/CloudVE/galaxy-kubernetes/pull/34/commits/c072c8b70c00a7f1a444b98dd8b56fad25823217